postgres/tests/test_statements.py

```diff
@@ -87,7 +88,7 @@ def test_statement_metrics_multiple_pgss_rows_single_query_signature(
     # don't need samples for this test
     dbm_instance['query_samples'] = {'enabled': False}
     dbm_instance['query_activity'] = {'enabled': False}
-    dbm_instance['query_metrics'] = {'enabled': True, 'run_sync': True, 'incremental_query_metrics': True}
+    dbm_instance['incremental_query_metrics'] = True
```

Comment:
this looks wrong. why is `incremental_query_metrics` moved to the top level?

Reply:
oh yeah should be 
`dbm_instance['query_metrics']['incremental_query_metrics'] = True`
thx